build.js

```diff
@@ -35,11 +35,9 @@ function createFiles(platform) {
   // first do a rollup of all tokens
   let files = formats[platform].map((format) => createFile(format, 'index'));
 
-  // console.log(files);
-
   // then, for each category, output a file in each format
   Object.keys(properties).forEach(function(category) {
-    let filter = (prop) => prop.attributes.category == category;
+    let filter = (prop) => prop.attributes.category === category;
```

Comment:
Boyscouting! Didn't see a reason for loose equality here.
Generally prefer strict equality over loose equality unless there's an explicit reason for it. 

If there is a reason though, happy to revert this change!

Reply:
No reason, just an oversight 👍 good catch